ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -126,6 +126,20 @@ tests:
     - ref: stackrox-e2e
   timeout: 5h0m0s
 - as: openshift-oldest-qa-e2e-tests
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 5h0m0s
+    version: "4.7"
+  optional: true
+  run_if_changed: ^((generated|image|operator|pkg|make|deploy|scripts)/.*|Makefile|status\.sh|EXPECTED_GO_VERSION)
+  steps:
+    test:
+    - ref: stackrox-e2e
+  timeout: 5h0m0s
+- as: openshift-ocp-4-8-qa-e2e-tests
```

Comment:
Needed to change the test name as ocp 4.8 is no longer the oldest. Tried openshift-second-oldest-qa-e2e-tests, but the job name was too long and OSCI rejected it.

Reply:
I would drop the 'openshift' i.e. just `ocp-4-8-qa-e2e-tests`.

Reply:
Done in the other PRs